service/http/dffml_service_http/routes.py

```diff
@@ -196,15 +292,45 @@ async def multicomm_dataflow(self, config, request):
                 results = {
                     str(ctx): result async for ctx, result in octx.run(*inputs)
                 }
-                # TODO Implement input and presentation stages?
-                """
-                if config.presentation == "blob":
-                    return web.Response(body=results)
-                elif config.presentation == "text":
-                    return web.Response(text=results)
+                if config.output_mode == "json":
+                    return web.json_response(results)
+
+                # content_info is a list in case of stream
+                # and string in others
+                postprocess_mode, *content_info = config.output_mode.split(":")
+
+                if postprocess_mode == "stream":
+                    # stream:text/plain:get_single.beef
+                    content_type, output_keys = content_info
+                    output_data = traverse_get(
+                        results, *output_keys.split(".")
+                    )
+
+                    response = web.StreamResponse(
+                        status=200, headers={"Content-Type": content_type}
+                    )
+
+                    await response.prepare(request)
+
+                    for data in output_data:
+                        await response.write(data)
+                    await response.write_eof()
+                    return response
+
+                output_data = traverse_get(
+                    results, *content_info[0].split(".")
+                )
+
+                if postprocess_mode == "bytes":
+                    return web.Response(body=output_data)
```

Comment:
We should probably require a content type be set for when the mode is bytes too. So, another: `content_type, output_keys = content_info`